src/lib/seam/connect/unstable/models/devices/managed-device.ts

```diff
@@ -21,6 +21,19 @@ export const common_device_properties = z.object({
     .describe(
       'Name of the device. Enables administrators and users to identify the device easily, especially when there are numerous devices.',
     ),
+  nickname: z
+    .string()
+    .optional()
```

Comment:
@mc2147 Subtle, but I think we want this to be null not undefined since it applies to all devices but is null by default. Do you think it applies here? https://github.com/seamapi/rfc/blob/main/rfcs/2023-03-01-api-design.md#22-null-vs-undefined

I noticed a similar case for user identities with phone number and email (and ACS users, if the 3pp supports those fields).